### PR TITLE
[WIP] Readme revisions according to #7

### PR DESCRIPTION
**Pull request by awalvie**
The PR serves as a way for all of us to review and revise the README before the final submission of out project and solving #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,71 @@
 # CovidAid
+
+## The elevator pitch
+A mobile application where people can make requests asking for help, and other members of the community close to them can accept any request and provide help.
+
 ## Inspiration
-Social distancing and lockdown has become the necessary evil in this global pandemic situation, and everyone is struggling getting basic day to day things.But this dilemma becomes a lot more serious for the older people and the COVID patients who are living alone in small cities, where you don't have grocery delivery apps, or medicine delivery apps. In these bad times, people normally tend to depend on community for help,and that's where *CovidAid* steps in :bulb: to ease the situation.
+The idea stems from an experience a fellow teammate had. His neighbors got infected with COVID19 and 3 out of the 4 members of the household had to be admitted to a hospital at a different location, leaving the oldest member of the household alone and without the ability to ask for help from the community (because he was supposed to be in isolation) in case he needed anything.
```
Comment (utsavcoding) [inline README.md:7]: I think as per Nitish doc, we can skip the situational story, in the first paragraph. And also mention the fact that in small cities, you dont have startups like food delivery apps, or medicine delivery apps.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,71 @@
...
-1. **Ask for help** - This is for help-seekers, who can put forth their request using the app with all relevant details,which will be posted to the timeline as a live request.
-2. **Like to help** - This is for the helpers, who can choose to pick up any request as per their convenience from the list of posted live requests on timeline.
+CovidAid is a community-driven platform for the people. On it, people looking to help others can locate people that need help and fulfill their request with mutual consent, based on their location and proximity. Every user can do either of two things:
+1. **Ask for help** - This is for help-seekers. They can make requests by filling out details of what they would like help with and each new request is then posted onto a timeline for everyone to see.
+2. **Like to help** - This is for people who would like to help. They will be presented with a number of active requests, which they can look through and accept as they see fit.
+
+For each request they complete, helpers will score points and the top helpers can be seen on the leaderboard.
```
Comment (utsavcoding) [inline README.md:16]: This looks fine, just mention the "kindness points" instead of "score points".

**Review by utsavcoding — CHANGES_REQUESTED** [review]
@NitishGadangi @awalvie Over to you guys!

File: README.md [hunk 3]
```diff
@@ -1,25 +1,71 @@
 # CovidAid
```
Comment (utsavcoding) [inline README.md:1]: @NitishGadangi You can put a logo in this section

**Review by utsavcoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (awalvie) [reply README.md:16]: Kindness points feels a little too cliche

File: README.md (hunk 1 above)
Comment (awalvie) [reply README.md:7]: Hmm, are you sure? That's where the idea came from though. The small city problem makes sense, I'll add that in.

Comment (utsavcoding) [reply README.md:7]: I think we can keep the story for the video. Just mention the inspiration over here and keep it to the point.

File: README.md (hunk 2 above)
Comment (utsavcoding) [reply README.md:16]: Yeah, I thought we were naming it kindness points from the meeting we had(Nitish mentioned it as kindness points). I checked the app right now. Its just points. So I think thats fine. Keep it score points.

**Review by utsavcoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (awalvie) [reply README.md:7]: alright, that makes sense.

**Pull request merged**